packages/hyperion-autologging/src/ALElementInfo.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates. All Rights Reserved.
+ */
+
+'use strict';
+
+import { ComponentNameValidator, defaultComponentNameValidator, getReactComponentData_THIS_CAN_BREAK } from './ALReactUtils';
+import type { ReactComponentData } from './ALReactUtils';
+import { getVirtualPropertyValue, setVirtualPropertyValue } from '@hyperion/hyperion-core/src/intercept';
+
+const AL_ELEMENT_INFO_PROPNAME = '__alInfo';
+const AUTO_LOGGING_COMPONENT_TYPE = 'data-auto-logging-component-type';
+
+export default class ALElementInfo {
+  element: Element;
+  _reactComponentData: ReactComponentData | null = null;
```

Comment:
in TS, there is no "convention" for private / public members of class. If you want these to be private, you should explicitly say that using "**private**" keyword.